puppet/zulip_ops/manifests/postgres_common.pp

```diff
@@ -6,20 +6,21 @@
                                  "pv",
                                  "python3-pip",
                                  "python-pip",
-                                 # "python3-gevent", # missing on trusty
-                                 "python-gevent",
                                  # Postgres Nagios check plugin
                                  "check-postgres",
                                  ]
   package { $internal_postgres_packages: ensure => "installed" }
 
+  exec {"pip2_ensure_latest":
+    command => "/usr/bin/pip2 install -U pip==9.0.1",
+    creates => "/usr/local/bin/pip2",
+    require => Package['python-pip'],
+  }
```

Comment:
Why do we need the pip upgrade?

Reply:
Because 9.0+ on Python 2 is recommended by www.python3statement.org/practicalities/.